src/pages/journal/journal-components/journal-candidate/journal-candidate.scss

```diff
@@ -0,0 +1,32 @@
+journal-candidate {
+    & > div {
+      height: 100%;
+      padding: 0.3em 0.75em;
+      background-color: map-get($colors-gds, gds-grey-4);
+      border-radius: 13px;
+      @include box-shadow(0px, 2px, 4px, rgba(0, 0, 0, 0.55));
+      div {
+        height: 50%;
```

Comment:
Is this div height used? I can only see the top level div.

Reply:
removed the div from the sass.